cmd/termui/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -263,6 +269,26 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
 	wr.chainInfo.Rows = rows
 }
 
+func computeRedundancyStr(redundancyLevel uint64, redundancyIsMainActive string) string {
+	if redundancyIsMainActive == statusNotApplicable {
+		return ""
+	}
+
+	redundancyStr := "Redundancy: "
+	if redundancyLevel < 0 {
+		redundancyStr += "inactive"
+	} else {
+		if redundancyLevel == 0 {
+			redundancyStr += "Main machine"
```

Comment:
lower case first letter? to be similar to `back-up ...`

Reply:
fixed